### DISCUSSION TITLE
Can we setup mimir on local minikube

### DISCUSSION THREAD

**Discussion by sivamalla42** [grafana/mimir#5677] [Help and support]
Hey all,
I am exploring mimir, I want to setup mimir on local minikube, did anyone tried this option or any blogs if we have, please do share
Thanks

**Comment by majerus1223** [comment]
I did this on rancher desktop, similar to minikube.. start with the small chart remove all the tolerations and anti affinity rules along with resources. It should start up but was a huge pain

**Comment by aggholamyy** [comment]
I tried it at first it failed,
solution to my case:

- increase resource allocated by docker (on macos you need to use docker desktop)
- increase resource allocated by minikube 

`minikube stop`
`minikube config set memory 12288   # Set 12GB memory`
`minikube config set cpus 6        # Set 6 CPUs`
`minikube start` 
I didn't change anything on helm chart.
